examples/pytorch/rgcn/link_utils.py

```diff
@@ -0,0 +1,280 @@
+"""
+Utility functions for link prediction
+Most code is adapted from authors' implementation of RGCN link prediction:
+https://github.com/MichSchli/RelationPrediction
+
+"""
+import numpy as np
+import torch as th
+import dgl
+
+# Utility function for building training and testing graphs
+
+def comp_deg_norm(g):
```

Comment:
I feel this is also quite common. I could definitely see Transform module should include it.

Reply:
They are actually equivalent. Replaced.